video/out/wayland_common.c

```diff
@@ -3300,12 +3308,14 @@ bool vo_wayland_init(struct vo *vo)
      * before mpv does anything else. */
     wl_display_roundtrip(wl->display);
 
+#if HAVE_WAYLAND_PROTOCOLS_1_41
     // Only bind to vo_dmabuf_wayland for now to avoid conflicting with VK_hdr_layer
```

Comment:
Ah just remembered this comment. It won't be accurate since mesa will be the one doing this eventually. Maybe just `Only bind to vo_dmabuf_wayland to avoid conflicting with graphics drivers`